.github/CODEOWNERS

```diff
@@ -0,0 +1,22 @@
+# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+*           @status-im/desktop                              # By default adding the desktop team to review
+Makefile    @jakubgs @yakimant @status-im/desktop
+/ci/        @jakubgs @yakimant                              # Any change in the ci dir requires approval from Jakub or Anton
+/nix/       @jakubgs @yakimant
```

Comment:
Why not use use `@status-im/devops` instead of me and Anton explicitly?
Also, we might want to include the `/scripts/` folder as well.